.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Report a bug in this library
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!--
+This form is for bug reports for this library only.
+For general, language-agnostic questions and troubleshooting, please ask/look for answers [here](INSERT LINK TO FORUM).
```

Comment:
replace link with https://groups.google.com/g/adwords-api

Reply:
Done. Also updated the language question.